src/fuselage/CCPACSFuselageProfiles.cpp

```diff
@@ -109,4 +109,5 @@ CCPACSFuselageProfile& CCPACSFuselageProfiles::GetProfile(int index) const
     return static_cast<CCPACSFuselageProfile&>(*m_fuselageProfiles[index]);
 }
 
+
```

Comment:
Let's not commit white space as part of this change.

```suggestion
```

Reply:
I recently made good experience with [this Wiki description on `Automatic code formatting to TiGL style guide`](https://github.com/DLR-SC/tigl/wiki/TiGL-Programmers-Guide#automatic-code-formatting-to-tigl-style-guide). 

Clang works fine with the proposed settings in VSCode in my case.

Reply:
Thanks for the hint! I have to admit that I haven't setup automatic style checking myself for a while, but it can be really helpful.